OrbitGl/FunctionsDataView.cpp

```diff
@@ -218,11 +218,12 @@ std::wstring FUN_CREATE_RULE = L"Create Rule";
 std::wstring FUN_SET_AS_FRAME = L"Set As Main Frame";
 
 //-----------------------------------------------------------------------------
-std::vector<std::wstring> FunctionsDataView::GetContextMenu(int a_Index) {
+std::vector<std::wstring> FunctionsDataView::GetContextMenu(
+    int a_ClickedIndex, const std::vector<int>& a_SelectedIndices) {
```

Comment:
I suggest changing names to conform with styleguide naming conventions, since you are changing these anyways.

Reply:
See comment in main discussion.